docs/resources/Stage_Instance.md

```diff
@@ -0,0 +1,70 @@
+# Stage Instance Resource
+
+A _Stage Instance_ holds information about a live stage.
+
+## Definitions
+
+Below are some definitions related to stages.
+
+- **Liveness:** A Stage channel is considered _live_ when there is an associated stage instance. Conversely, a Stage channel is _not live_ when there is no associated stage instance.
+- **Speakers:** A participant of a Stage channel is a _speaker_ when their [voice state](#DOCS_RESOURCES_VOICE/voice-state-object)
+  is not `suppress`ed, and has no `request_to_speak_timestamp`.
+- **Moderators**: A member of the guild is a _moderator_ of a Stage channel if they have all of the following [permissions](#DOCS_TOPICS_PERMISSIONS/permissions):
+  - `MANAGE_CHANNELS`
+  - `MUTE_MEMBERS`
+  - `MOVE_MEMBERS`
+
+## Auto Closing
+
+When a Stage channel has no speakers for a certain period of time (on the order of minutes) it will be automatically deleted.
```

Comment:
For some extra color, this should be on average 10-15 minutes. The exact time is an implementation detail and will not be documented.

The goal for us here is to not have lingering stage instances with basically no one in them. We found that for large Stages people just go AFK. In the future, we plan that ending a stage instance will kick everyone out of the stage.